src/redux/weather.js

```diff
@@ -0,0 +1,43 @@
+import { createAction, createAsyncThunk, createReducer } from '@reduxjs/toolkit';
+
+const FETCH_WEATHER = 'FETCH_WEATHER';
+const PICK_CITY = 'PICK_CITY';
+
+const cities = ['Paris', 'Barcelona', 'Copenhagen', 'Vienna', 'Berlin', 'Lisbon', 'Madrid', 'New York', 'Buenos Aires', 'Montevideo', 'Kampala', 'Budapest', ' Beijing', 'Tripoli', 'Kurkata'];
+
+const fetchoneCityWeather = async (city) => {
+  const resp = await fetch(`https://weather.visualcrossing.com/VisualCrossingWebServices/rest/services/timeline/${city}/2023-02-17?unitGroup=metric&include=days&key=9XBYDAN3EHGMRUXXB7VHBZYFV&contentType=json`);
```

Comment:
- [ ] [OPTIONAL] Just like the previous reviewer has opined, please note that it is against best practices to expose your API keys as you have. Please ensure to use environment variables instead to protect your API keys 👍